### DISCUSSION TITLE
Is fix for CVE-2023-4863 provided

### DISCUSSION THREAD

**Discussion by ivan-borodiei** [dlemstra/Magick.NET#1470] [General] [answered]
Just wondering if a fix for CVE-2023-4863 has been provided?
If so, which version should we use.

https://www.upguard.com/blog/libwebp-cve-2023
https://nvd.nist.gov/vuln/detail/CVE-2023-4863
https://arcticwolf.com/resources/blog/cve-2023-4863/

**Comment by dlemstra** [comment, marked as answer]
I have updated the release notes of this version https://github.com/dlemstra/Magick.NET/releases/tag/13.3.0 that includes a fix for that CVE.

**Reply by ivan-borodiei** [reply]
Thank you very much!
